src/components/App.tsx

```diff
@@ -131,6 +131,7 @@ type AppState = {
     debug: boolean
   }
   fileHandle: FileSystemFileHandle | null
+  file: PMTiles | null
```

Comment:
This doesn't look like a good name for the relevant state...

Reply:
how about localPMTiles?